src/test/ui/async-await/issues/issue-78938-async-block.stderr

```diff
@@ -0,0 +1,30 @@
+error[E0373]: async block may outlive the current function, but it borrows `room_ref`, which is owned by the current function
+  --> $DIR/issue-78938-async-block.rs:8:39
+   |
+LL |       let gameloop_handle = spawn(async {
+   |  _______________________________________^
+LL | |         game_loop(Arc::clone(&room_ref))
+   | |                               -------- `room_ref` is borrowed here
+LL | |     });
+   | |_____^ may outlive borrowed value `room_ref`
+   |
+note: function requires argument type to outlive `'static`
+  --> $DIR/issue-78938-async-block.rs:8:33
+   |
+LL |       let gameloop_handle = spawn(async {
+   |  _________________________________^
+LL | |         game_loop(Arc::clone(&room_ref))
+LL | |     });
+   | |_____^
```

Comment:
Ideally we would change or remove this note when caused by `async` blocks, particularly when we suggest `async move`...

Reply:
I've removed this in my latest commit.